pegasus/sites.v3/code.org/public/educate/professional-learning/events.md

```diff
@@ -1,12 +1,12 @@
 ---

```

Comment:
links run off the page: 
<img width="320" alt="screen shot 2017-04-03 at 1 47 28 pm" src="https://cloud.githubusercontent.com/assets/12300669/24631143/25900bb0-1874-11e7-8da4-6a0d030b4e20.png">